mvc/authorizationProcess.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+final class authorizationProcess
+{
+    public function auth()
+    {
+        $email = filter_var(trim($_POST['Email']), FILTER_SANITIZE_STRING);
```

Comment:
```suggestion
        $email = htmlspecialchars(trim($_POST['Email']));
```

And add validation for email format